src/content/reference/react/use.md

```diff
@@ -0,0 +1,769 @@
+---
+title: use
+---
+
+<Wip>
+This section is incomplete. See also the [draft use RFC](https://github.com/acdlite/rfcs/blob/first-class-promises/text/0000-first-class-support-for-promises.md#usepromise).
+</Wip>
+
+[//]: # (TODO: replace the `<Note></Note>` below with `<Canary></Canary>` once https://github.com/reactjs/react.dev/pull/6173 is merged)
+<Note>
+The `use` hook is only supported when using [React Server Components](/learn/start-a-new-react-project#bleeding-edge-react-frameworks). React Server Components are only available in React’s canary and experimental channels. Learn more about [React's release channels here](/community/versioning-policy#all-release-channels). 
+</Note>
+
+<Intro>
+
+`use` is a React Hook that lets you read the value of a resource like a [promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) or [context](/learn/passing-data-deeply-with-context).
+
+```js
+const value = use(resource);
+```
+
+</Intro>
+
+<InlineToc />
+
+---
+
+## Reference {/*reference*/}
+
+### `use(resource)` {/*use*/}
+
+Call `use` in your component to read the value of a resource like a [promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) or [context](/learn/passing-data-deeply-with-context).
+
+```jsx
+import { use } from "react";
+
+function Message({ messagePromise }) {
+  const message = use(messagePromise);
+  const theme = use(ThemeContext);
+  //...
+```
+
+Unlike all other React Hooks, the `use` Hook can be called within conditional statements like loops and `if` statements. Like other React Hooks, the parent function of a Hook must be a React component or Hook.
+
+A component that passes a promise to `use` integrates with [`Suspense`](/reference/react/Suspense) and [`ErrorBoundary`](/reference/react/Component#catching-rendering-errors-with-an-error-boundary). When a pending promise is passed to `use`, the fallback of the closest Suspense Boundary will be displayed. When the promise is resolved, the component calling `use` will be rendered using the data from the resolved promise. The rendered component will then replace the Suspense Boundary fallback. If the promise passed to `use` is rejected, the fallback of the nearest Error Boundary will be displayed. 
```

Comment:
> A component that passes a promise to use integrates with Suspense and ErrorBoundary.

this is very passive voice, and it's also confusing to me because it makes a statement about the user's components that at a glance could be read as a requirement that the user must fulfill. maybe just "When called with a promise, the `use` Hook integrates with"?

looks like house style is `Suspense boundary` and `error boundary`

I also think this description isn't quite right because it doesn't account for startTransition.

Reply:
Took your suggested fix for passive voice and fixed the Suspense boundary and error boundary.

I reworded to the explanation to remove the problematic reference to rendering. I think that is what you were referring to what you said the description isn't consistent with `startTransition`?  What do you think?